source/base/utilities.cc

```diff
@@ -13,6 +13,11 @@
 //
 // ---------------------------------------------------------------------
 
+#include <deal.II/base/config.h>
+
+#ifdef DEAL_II_MSVC
+#  include <winsock2.h>
+#endif
```

Comment:
This fixes conflict between winsock.h and winsock2.h
```
2>  C:\Program Files (x86)\Windows Kits\10\Include\10.0.14393.0\um\winsock.h(1011): note: see declaration of 'sockaddr_in'
2>C:\Program Files (x86)\Windows Kits\10\Include\10.0.14393.0\um\winsock2.h(136): error C2011: 'fd_set': 'struct' type redefinition
```

intel tbb includes winsock.h somewhere so we should include winsock2.h before it to avoid conflict